gui/default/assets/css/tree.css

```
@@ -56,3 +56,91 @@
     white-space: normal;
     word-break: break-all;
 }
+
+.fancytree-temp .fancytree-title {
+    opacity: 0.6;
```

```suggestion
    color: #626262
```

The opacity makes the contrast ratio fail WCAG AA, so I would suggest to use these colours instead. The value above is for the light theme. The values for the black and dark themes are `color: #aaa`. All these are already used and come from `help-block`.